draft-gazdag-x509-hash-sigs.md

```diff
@@ -108,43 +136,64 @@ lists.
 
 Hash-Based Signature (HBS) Schemes combine Merkle trees with One/Few Time
 Signatures (OTS/FTS) in order to provide digital signature schemes that remain
-secure even when quantum computers become available. There security is well
+secure even when quantum computers become available. Their security is well
 understood and depends only on the security of the underlying hash function. As
 such they can serve as an important building block for quantum computer
 resistant information and communication technology.
 
-The private key of HSS, XMSS and XMSS^MT is a finite collection of OTS keys,
-hence only a limited number of messages can be signed and the private key's
-state must be updated and persisted after signing to prevent reuse of OTS keys.
-Due to thise statefulness of the private key and the limited number of
-signatures that can be created, these signature algorithms might not be
-appropriate for use in interactive protocols. While the right selection of
-algorithm parameters would allow a private key to sign a virtually unbounded
-number of messages (e.g. 2^60), this is at the cost of a larger signature size
-and longer signing time. Since these algorithms are already known to be secure
-against quantum attacks, and because roots of trust are generally long-lived and
-can take longer to be deployed than end-entity certificates, these signature
-algorithms are more appropriate to be used in root and subordinate CA
-certificates. They are also appropriate in non-interactive contexts such as code
-signing. In particular, there are multi-party IoT ecosystems where publicly
-trusted code signing certificates are useful.
+The private key of HSS, XMSS and XMSS^MT is a finite collection of OTS keys, hence only a limited
+number of messages can be signed and the private key's state must be updated and persisted
+after signing to prevent reuse of OTS keys. Due to this statefulness of the private key
+and the limited number of signatures that can be created, these signature algorithms might
+not be appropriate for use in interactive protocols. While the right selection of algorithm
+parameters would allow a private key to sign a virtually unbounded number of messages (e.g. 2^60),
+this is at the cost of a larger signature size and longer signing time. Since these algorithms
+are already known to be secure against quantum attacks, and because roots of trust are generally
+long-lived and can take longer to be deployed than end-entity certificates, these signature
+algorithms are more appropriate to be used in some use case scenarios.
```

Comment:
```suggestion
The private key of HSS, XMSS and XMSS^MT is a finite collection of OTS keys, hence only a limited
number of messages can be signed and the private key's state must be updated and persisted after signing to prevent reuse of OTS keys. While the right selection of algorithm parameters would allow a private key to sign a virtually unbounded number of messages (e.g. 2^60), this is at the cost of a larger signature size and longer signing time. Due to the statefulness of the private key of HSS, XMSS and XMSS^MT and the limited number of signatures that can be created, these signature algorithms might not be appropriate for use in interactive protocols. However, in some use case scenarios the deployment of these signature algorithms may be appropriate. Such use cases are described and discussed later in {{use-cases-hbs-x509}}.
```